__tests__/app.test.js

```diff
@@ -39,21 +39,19 @@ describe("GET /api/reviews/:review_id", () => {
 			.get("/api/reviews/10")
 			.expect(200)
 			.then(({ body: { review } }) => {
-				const output = [
-					{
-						review_id: 10,
-						title: "Build you own tour de Yorkshire",
-						designer: "Asger Harding Granerud",
-						owner: "mallionaire",
-						review_img_url:
-							"https://images.pexels.com/photos/258045/pexels-photo-258045.jpeg?w=700&h=700",
-						review_body:
-							"Cold rain pours on the faces of your team of cyclists, you pulled to the front of the pack early and now your taking on exhaustion cards like there is not tomorrow, you think there are about 2 hands left until you cross the finish line, will you draw enough from your deck to cross before the other team shoot passed? Flamee Rouge is a Racing deck management game where you carefully manage your deck in order to cross the line before your opponents, cyclist can fall slyly behind front runners in their slipstreams to save precious energy for the prefect moment to burst into the lead ",
-						category: "social deduction",
-						created_at: "2021-01-18T10:01:41.251Z",
-						votes: 10,
-					},
-				];
+				const output = {
+					review_id: 10,
```

Comment:
Bear in mind, should you make any changes to the data for this article, the values may change. The only one that will remain consistant always is the review_id. I would therefore suggest using a flexible matcher here such as expect.objectContaining or .toMatchObject. and only checking the article Id is 10.